src/components/common/header/header.jsx

```diff
@@ -7,28 +8,48 @@ import './header.scss';
 import logo from '../../../images/nouri-logo.png';
 
 export const Header = () => {
-
+  
   const [navClass, toggleNav] = useState("hide");
-
+  const [POPUPDISPLAY, popUp] = useState('popupdisplayHIDE'); 
+  //const [submitEmail, name, email] = useState('name', 'email');  
   const toggleResponsiveNav = () => {
     if (navClass === "mobile") {
       toggleNav("hide");
     } else {
       toggleNav("mobile");
     }
+  }   
+
+  //fn handleSubmitEmail is not used, could be used later
+  // const handleSubmitEmail = (e)=>{
+  //    //this fn is never run and can be used in the future 
+  //    submitEmail(e.target.name.value, e.target.email.value) 
+  //    console(e, name, email )
+
+  //  } 
```

Comment:
No need to keep unused code in comment for just in case.

Reply:
👍